main.js

```diff
@@ -15,12 +15,27 @@ var createdActivities = [];
 
 categoryContainer.addEventListener("click", activateIcon);
 buttonStartActivity.addEventListener('click', submitForm);
+inputMinutes.addEventListener("keyup", validateNumberMinutes)
+inputSeconds.addEventListener("keyup", validateNumberSeconds)
+
 
 
 function activateIcon() {
   displayActivatedIcon(event);
 }
 
+function validateNumberMinutes(e){
+  if (!(e.keyCode >= 48 && e.keyCode <= 57) && !(e.keyCode >= 96 && e.keyCode <= 105) && !(e.keyCode == 8)){
+    e.target.value = e.target.value.substring(0, e.target.value.length - 1);
+  };
+};
+
+function validateNumberSeconds(e){
+  if (!(e.keyCode >= 49 && e.keyCode <= 57) && !(e.keyCode >= 96 && e.keyCode <= 105) && !(e.keyCode == 8)){
+    e.target.value = e.target.value.substring(0, e.target.value.length - 1);
+  };
+};
+
```

Comment:
We need a helper function to pass these to the event listeners

Reply:
Good eye, added the helper functions.